code/game/gamemodes/nuclear/nuclearbomb.dm

```diff
@@ -14,29 +14,29 @@ GLOBAL_VAR(bomb_set)
 	icon_state = "nuclearbomb0"
 	density = 1
 	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
-	var/deployable = 0
-	var/extended = 0
-	var/lighthack = 0
+	var/extended = FALSE
+	var/lighthack = FALSE
 	var/timeleft = 120
-	var/timing = 0
+	var/timing = FALSE
+	var/exploded = FALSE
 	var/r_code = "ADMIN"
-	var/code = ""
-	var/yes_code = 0
-	var/safety = 1
+	var/code
+	var/yes_code = FALSE
+	var/safety = TRUE
 	var/obj/item/disk/nuclear/auth = null
 	var/removal_stage = NUKE_INTACT
 	var/lastentered
-	var/is_syndicate = 0
+	var/is_syndicate = FALSE
 	use_power = NO_POWER_USE
 	var/previous_level = ""
 	var/datum/wires/nuclearbomb/wires = null
 
 /obj/machinery/nuclearbomb/syndicate
-	is_syndicate = 1
+	is_syndicate = TRUE
 
 /obj/machinery/nuclearbomb/New()
 	..()
-	r_code = "[rand(10000, 99999.0)]"//Creates a random code upon object spawn.
+	r_code = rand(10000, 99999.0) // Creates a random code upon object spawn.
```

Comment:
Do not change this from a string to an integer, unless you want to edit it in other places in the codebase. Heres one for example:
`code\game\gamemodes\blob\blob_report.dm` Line `21`
```dm
			var/nukecode = "[rand(10000, 99999)]"
			for(var/obj/machinery/nuclearbomb/bomb in GLOB.machines)
				if(bomb && bomb.r_code)
					if(is_station_level(bomb.z))
						bomb.r_code = nukecode
```